doc/bc/changes-6.0.md

```diff
@@ -211,6 +211,13 @@ Changes affecting version compatibility with former or future versions.
   Those need to be changed to custom content view controllers, that use a contentId instead of a locationId as an
   argument. The location is available in the `$parameters` array.
 
+* The `viewBlock` and `viewBlockById` `PageController` actions are deprecated.
+  `viewAction` must be used instead. It accepts both `block` and `id` as parameters.
```

Comment:
should we mention `block` here? isn't that handled by parameter converter? *as in, wasn't that a bug some years ago, people passing object as param*